doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1458 @@
+    The Kconfig option enables support for the derivation function SP 800-108r1 CMAC in counter mode, which is supported by the nrf_cracen driver.
+
+* :ref:`trusted_storage_readme` library:
+
+  * Added support for Zephyr Memory Storage (ZMS), as an alternative to the NVS file system.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* Added semantic version support to :ref:`nrf_desktop_config_channel_script` Python script for devices that use the SUIT DFU.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``a4eda30f5b0cfd0cf15512be9dcd559239dbfc91``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added :ref:`Compressed firmware update support <mcuboot_image_compression>`.
+
+* Fixed:
+
+  * An issue with wrong network core address checking in MCUboot for nRF5340 network core firmware updates.
+  * An issue with MCUboot updates in MCUboot when a swap mode was selected, which could lead to possible data corruption by implementing additional NSIB-specific (overwrite-only) upgrade mode for sysbuild.
+  * An multiple issues with checking firmware update slot selection in MCUboot for sysbuild.
+  * An issue with MCUboot cleanup unused slots feature, which could lead to possible empty firmware update of the main image and bricking the device for sysbuild.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``ea02b93eea35afef32ebb31f49f8e79932e7deee``, with some |NCS| specific additions.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline ea02b93eea ^23cf38934c
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^ea02b93eea
+
+The current |NCS| main branch is based on revision ``ea02b93eea`` of Zephyr.
+
+.. note::
+   For possible breaking changes and changes between the latest Zephyr release and the current Zephyr version, refer to the :ref:`Zephyr release notes <zephyr_release_notes>`.
```

Comment:
Any additions specific to NCS?

Reply:
@rlubos would you mind checking this section as part of the release work?

Reply:
The SHAs look like a copy-paste from the previous release, I've left comments inline with correct ones.